docker/run.sh

```diff
@@ -72,8 +72,8 @@ reload)
     bundle exec rake db:create 
     echo "executing rake db:schema:load"
     bundle exec rake db:schema:load
-    echo "executing rake db:data:load"
-    bundle exec rake db:data:load
+    echo "executing rake claims:restore"
+    bundle exec rake claims:restore
     ;;
 reseed)
     # db:clear
```

Comment:
Absolutely not a problem don't worry. it's just a minimal change. Please let me know if I need to close this, i will leave it for the time being.